packages/sshnoports/pubspec.yaml

```diff
@@ -32,3 +30,5 @@ dev_dependencies:
   lints: ^2.1.1
   test: ^1.24.7
   mocktail: ^0.3.0
+  build_runner: ^2.4.6
+  build_version: ^2.1.1
```

Comment:
For automated version matching, version.dart will now automatically update to the value in pubspec.yaml